security/pkg/nodeagent/cache/secretcache_test.go

```diff
@@ -44,7 +45,8 @@ func TestGetSecret(t *testing.T) {
 	}()
 
 	proxyID := "proxy1-id"
-	gotSecret, err := sc.GetSecret(proxyID, fakeSpiffeID, "jwtToken1" /*jwtToken*/)
+	ctx := context.Background()
+	gotSecret, err := sc.GetSecret(ctx, proxyID, fakeSpiffeID, "jwtToken1" /*jwtToken*/)
```

Comment:
I think "/*jwtToken*/" is unnecessary here :)

Reply:
removed.